projects/bp-gallery/src/components/common/picture-gallery/PictureGrid.scss

```diff
@@ -42,3 +42,38 @@
     }
   }
 }
+
+.selection-buttons {
+  margin-top: 1.5rem;
+  background: #e5e5e5;
+  border-radius: 0.3rem;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 1rem;
+  gap: 10px;
+
+  .MuiButton-root {
+    background-color: $primaryColor;
+  }
+}
```

Comment:
This doesn't seem to be used anywhere (I think it has come from bulk edit somehow, since git blame says so, but I don't know why)